src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/personalisation/appellant/letter/AppellantInternalLateRemissionRejectedLetterPersonalisation.java

```diff
@@ -51,9 +53,7 @@ public Map<String, String> getPersonalisation(Callback<AsylumCase> callback) {
                 .getCaseDetails()
                 .getCaseData();
 
-        YesOrNo isAppellantInUK = asylumCase.read(AsylumCaseDefinition.APPELLANT_IN_UK, YesOrNo.class).orElse(YesOrNo.NO);
-
-        List<String> appellantAddress = switch (isAppellantInUK) {
+        List<String> appellantAddress = switch (isAppellantInUK(asylumCase)) {
             case YES -> getAppellantAddressAsList(asylumCase);
```

Comment:
Same as above checkout 

`    public static Set<String> getAppellantAddressInCountryOrOoc(final AsylumCase asylumCase) {
`

Reply:
Removed the changes as it was for before the last merge that was done